src/components/molecules/Card/component.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from 'react';
+import { Props } from './props';
+import styled from '@emotion/styled';
+
+export const Card = styled.div<Props>``;
+
```

Comment:
Where are the styles for border? Border should exist if disableBorder prop is false (it's false by default).